<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayGame</class>
 <widget class="QDialog" name="PlayGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QRadioButton" name="rdbtnEasy">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>160</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Easy</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">btngrpDifficulty</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="rdbtnMedium">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>210</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Medium</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">btngrpDifficulty</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="rdbtnHard">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>240</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Hard</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">btngrpDifficulty</string>
   </attribute>
  </widget>
  <widget class="QTextEdit" name="txtName">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSaveName">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>90</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Store Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblName">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>80</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtNumber">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblWord">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>310</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtGuessedLetter">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>440</y>
     <width>104</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGuess">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>470</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Guess</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblCheck">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>530</y>
     <width>341</width>
     <height>61</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblScore">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>400</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Score: </string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdTimer">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>20</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="btngrpDifficulty"/>
 </buttongroups>
</ui>
